[debug] [2023-12-25T13:31:42.794Z] ----------------------------------------------------------------------
[debug] [2023-12-25T13:31:42.796Z] Command:       /usr/local/bin/node /Users/akk017/ProbeZone/caarya-founder-dictionary/node_modules/.bin/firebase login
[debug] [2023-12-25T13:31:42.796Z] CLI Version:   13.0.2
[debug] [2023-12-25T13:31:42.796Z] Platform:      darwin
[debug] [2023-12-25T13:31:42.797Z] Node Version:  v18.16.1
[debug] [2023-12-25T13:31:42.800Z] Time:          Mon Dec 25 2023 19:01:42 GMT+0530 (India Standard Time)
[debug] [2023-12-25T13:31:42.800Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-12-25T13:31:42.802Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[info] i  To change your data collection preference at any time, run `firebase logout` and log in again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=589946744&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2023-12-25T13:31:55.903Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2023-12-25T13:31:55.904Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2023-12-25T13:32:12.855Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2023-12-25T13:32:12.858Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2023-12-25T13:32:25.912Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------306654656079252664436786","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: getaddrinfo ENOTFOUND accounts.google.com
[debug] [2023-12-25T13:32:25.913Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (/Users/akk017/ProbeZone/caarya-founder-dictionary/node_modules/firebase-tools/lib/apiv2.js:246:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[debug] [2023-12-25T13:32:25.923Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2023-12-25T13:32:25.924Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"c254b618-8f35-444e-a9b8-c84bdc58fc0e"}
[debug] [2023-12-25T15:51:34.426Z] ----------------------------------------------------------------------
[debug] [2023-12-25T15:51:34.427Z] Command:       /usr/local/bin/node /Users/akk017/ProbeZone/caarya-founder-dictionary/node_modules/.bin/firebase login
[debug] [2023-12-25T15:51:34.428Z] CLI Version:   13.0.2
[debug] [2023-12-25T15:51:34.428Z] Platform:      darwin
[debug] [2023-12-25T15:51:34.428Z] Node Version:  v18.16.1
[debug] [2023-12-25T15:51:34.431Z] Time:          Mon Dec 25 2023 21:21:34 GMT+0530 (India Standard Time)
[debug] [2023-12-25T15:51:34.431Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-12-25T15:51:34.433Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[info] i  To change your data collection preference at any time, run `firebase logout` and log in again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=859155980&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2023-12-25T15:51:45.010Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2023-12-25T15:51:45.014Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2023-12-25T15:52:04.493Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2023-12-25T15:52:04.497Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2023-12-25T15:52:15.030Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------796965911335178509138822","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: getaddrinfo ENOTFOUND accounts.google.com
[debug] [2023-12-25T15:52:15.031Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (/Users/akk017/ProbeZone/caarya-founder-dictionary/node_modules/firebase-tools/lib/apiv2.js:246:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[debug] [2023-12-25T15:52:15.045Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2023-12-25T15:52:15.045Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"ef24da84-20cf-4e71-988d-8fdc71072f94"}
[debug] [2023-12-25T15:52:45.052Z] *** [apiv2] error from fetch(https://auth.firebase.tools/attest, {"headers":{},"method":"POST","body":"{\"session_id\":\"ef24da84-20cf-4e71-988d-8fdc71072f94\"}"}): FetchError: request to https://auth.firebase.tools/attest failed, reason: getaddrinfo ENOTFOUND auth.firebase.tools
[debug] [2023-12-25T15:52:50.076Z] Error: Timed out.
    at Timeout._onTimeout (/Users/akk017/ProbeZone/caarya-founder-dictionary/node_modules/firebase-tools/lib/utils.js:258:49)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
[error] 
[error] Error: An unexpected error has occurred.
